include/arch/arm/armv/armv8-a/64/armv/vcpu.h

```diff
@@ -619,7 +624,7 @@ static inline void vcpu_disable(vcpu_t *vcpu)
     /* Stage 1 MMU off */
     setSCTLR(SCTLR_DEFAULT);
     isb();
-    MSR(REG_HCR_EL2, HCR_NATIVE);
+    setHCR(HCR_NATIVE);
```

Comment:
the reorder should not have impact, but let see how HWrun goes.

Reply:
If the reorder doesn't have an impact, then could the intermediate `isb()` be removed?

Reply:
I can remove it, it's not present on arm_hyp, but don't know the implications